index.html

```diff
@@ -7,5 +7,20 @@
     <link rel="stylesheet" href="styles.css" />
     <title>Lenz' Quizz App</title>
   </head>
-  <body></body>
+  <body>
+    <section class="card">
+      <img class="bookmark" src="/images/bookmark.gif" alt="bookmark image" />
```

Comment:
Probably an image is not sufficient long term if you want to change the state (color) of the bookmark icon once a question was bookmarked. Maybe a SVG is more reasonable or try to create the shape whit a div in CSS - this would be my approach but not 100% sure if that is the way to do it.